sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusProcessorTest.java

```diff
@@ -415,6 +415,60 @@ public void testProcessorWithTracingEnabled() throws InterruptedException {
 
     }
 
+    @Test
+    public void testProcessorWithTracingEnabledWithoutDiagnosticId() throws InterruptedException {
+        final Tracer tracer = mock(Tracer.class);
+        final List<Tracer> tracers = Collections.singletonList(tracer);
+        final int numberOfTimes = 5;
```

Comment:
Overall looks good, just curious, why we need to send 5 times message, rather than just 1 time?

Reply:
I referred to the previous tests. Most of them send 5 or 10 times for test. I did the same times for safety. Send 1 time works too.